### PR TITLE
License our project as 3-clause BSD

### PR DESCRIPTION
**Pull request by miha-plesko**
A freaking huge PR since I had to add the copyright text to every recipe, but at least people will dare to use our software then.

### CODE REVIEW

File: docker_files/recipes/openfoam.core-2.4.0/openfoam-core.patch [hunk 1]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (gberginc) [inline docker_files/recipes/openfoam.core-2.4.0/openfoam-core.patch]: I suggest you do not use `#` in the patch file. According to patch file specs, everything above `---` is a comment.

File: docker_files/recipes/osv.bootstrap/bootstrap.patch [hunk 2]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (gberginc) [inline docker_files/recipes/osv.bootstrap/bootstrap.patch]: No `#` for patch file.

File: docker_files/recipes/osv.cli/cli.patch [hunk 3]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (gberginc) [inline docker_files/recipes/osv.cli/cli.patch]: No `#` for patch.

File: docker_files/recipes/osv.cloud-init/cloud-init.patch [hunk 4]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (gberginc) [inline docker_files/recipes/osv.cloud-init/cloud-init.patch]: No `#` for patch.

File: docker_files/recipes/osv.nfs/nfs.patch [hunk 5]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (gberginc) [inline docker_files/recipes/osv.nfs/nfs.patch]: No # for patch.

**Review by gberginc — CHANGES_REQUESTED** [review]
Check the comment about `patch` files. Otherwise thanks for this PR.

**Comment by miha-plesko** [comment]
Thanks for pointing out the unneeded use `#`, I've removed and repushed.

**Review by gberginc — APPROVED** [review]
Great @miha-plesko!

**Pull request merged**